src/Data/Skills/sup_dex.lua

```diff
@@ -410,6 +410,14 @@ skills["SupportArakaalisLustPlayer"] = {
 			label = "Arakaali's Lust",
 			incrementalEffectiveness = 0.054999999701977,
 			statDescriptionScope = "gem_stat_descriptions",
+			statMap = {
+				["support_debilitate_hit_damage_+%_final_per_poison_stack"] = {
+					mod("Damage", "MORE", nil, 0, 0, { type = "Multiplier", actor = "enemy", var = "PoisonStacks", limitVar = "ArakaaliMaxHitDamage", limitTotal = true } ),
```

Comment:
Needs to have ModFlag.Hit in the place of the first 0 after the nil so that the damage only scales hits